cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Do we need this file? it seems a little useless so we can probably cut it.

Reply:
Deleted it.